### PR TITLE
Convert to periodic job

### PR DESCRIPTION
**Pull request by dhiller**
Convert the testing from being PR based to regularly testing of latest master nightly.

Should then be visible by looking at https://prow.svc.ci.openshift.org/?repo=kubevirt%2Fkubevirt&type=periodic

### CODE REVIEW

**Comment by dhiller** [comment]
See kubevirt/project-infra#407 for context.

/cc @rmohr @danielBelenky

**Comment by dhiller** [comment]
See kubevirt/project-infra#407 for building nightly

File: ci-operator/config/kubevirt/kubevirt/kubevirt-kubevirt-master.yaml [hunk 1]
```diff
@@ -71,7 +64,7 @@ tests:
     TESTS_TO_FOCUS=$(grep -E -o '\[crit\:high\]' tests/*_test.go | sort | uniq | sed -E 's/tests\/([a-z_]+)\.go\:.*/\1/' | tr '\n' '|' | sed 's/|$//')
     FUNC_TEST_ARGS='--ginkgo.noColor --ginkgo.focus='"$TESTS_TO_FOCUS"' --ginkgo.regexScansFilePath=true --junit-output='"$ARTIFACT_DIR"'/junit.functest.xml' \
         bash -x ./hack/functests.sh
-
+  cron: 3 0 * * *
```
Comment (danielBelenky) [inline ci-operator/config/kubevirt/kubevirt/kubevirt-kubevirt-master.yaml:67]: I wonder if the time on our machines is synced with the time on GCP... if not, we need to adjust the times.
Also, I'm not sure that 45m are enough space between the triggers. Maybe give it 2 hours?

Comment (dhiller) [reply ci-operator/config/kubevirt/kubevirt/kubevirt-kubevirt-master.yaml:67]: Good point. Although I think all times on CRON taks should be UTC based, let's check this.

File: ci-operator/config/kubevirt/kubevirt/kubevirt-kubevirt-master.yaml [hunk 2]
```diff
@@ -71,7 +64,7 @@ tests:
     TESTS_TO_FOCUS=$(grep -E -o '\[crit\:high\]' tests/*_test.go | sort | uniq | sed -E 's/tests\/([a-z_]+)\.go\:.*/\1/' | tr '\n' '|' | sed 's/|$//')
     FUNC_TEST_ARGS='--ginkgo.noColor --ginkgo.focus='"$TESTS_TO_FOCUS"' --ginkgo.regexScansFilePath=true --junit-output='"$ARTIFACT_DIR"'/junit.functest.xml' \
         bash -x ./hack/functests.sh
-
+  cron: 2 3 * * *
```
Comment (dhiller) [inline ci-operator/config/kubevirt/kubevirt/kubevirt-kubevirt-master.yaml:67]: @danielBelenky I changed this now to start at 3AM, while the build job runs at 1AM.

**Pull request merged**